conformance_tests/sysman/test_sysman_init/src/test_init_sysman.cpp

```diff
@@ -24,4 +244,401 @@ TEST(SysmanInitTests,
   ASSERT_GT(pCount, 0);
 }
 
+TEST(SysmanInitTests, GivenSysmanInitializedThenCallingCoreInitSucceeds) {
+  ASSERT_EQ(ZE_RESULT_SUCCESS, zesInit(0));
+  uint32_t zesInitCount = 0;
+  ASSERT_EQ(ZE_RESULT_SUCCESS, zeInit(0));
+  uint32_t zeInitCount = 0;
+  ASSERT_EQ(ZE_RESULT_SUCCESS, zesDriverGet(&zesInitCount, nullptr));
+  ASSERT_GT(zesInitCount, 0);
+  ASSERT_EQ(ZE_RESULT_SUCCESS, zeDriverGet(&zeInitCount, nullptr));
+  ASSERT_GT(zeInitCount, 0);
```

Comment:
Why are all of these ASSERTS and not EXPECTs
ASSERTS are usually used to confirm pre-conditions

Reply:
Modified!